index.html

```diff
@@ -15,11 +15,12 @@
       <p>Use the left and right arrow to control the car!</p>
       <img src="./images/arrows.png" alt="" class="arrows-img" />
     </div>
-
     <div id="game-board">
-      <canvas id="canvas" width="500" height="700"></canvas>
+      <canvas id="canvas" width="500" height="700" style="border: 0cm;"></canvas>
+      <img src="/images/road.png" alt="" id="road-img" />
```

Comment:
If you're adding images to the canvas, you're gonna need to do it through javascript.
You did the correct logic on your main.js file, just missed a few steps that I'll show you on the next comment.